File: trunk/proj/CoreLibrary/src/com/jasonzqshen/familyaccounting/core/exception/format/DocumentIdentityFormatException.java
package com.jasonzqshen.familyaccounting.core.exception.format;

public class DocumentIdentityFormatException extends Exception {

	/**
	 * 
	 */
	private static final long serialVersionUID = 4457872286904353350L;

	
	public DocumentIdentityFormatException(String docIdStr) {
		super(String.format("Parsing document Identity \"%s\" contains error.",
				docIdStr));
	}
}

File: branches/V1.0/proj/WinStore_FamilyAccounting/WinStore_FamilyAccountingCore/Exceptions/FormatExceptions/DocumentIdentityFormatException.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace WinStore_FamilyAccountingCore.Exceptions.FormatExceptions
{
    public class DocumentIdentityFormatException : Exception
    {
        public DocumentIdentityFormatException(String docIdStr)
            : base(String.Format("Parsing document Identity \"{0}\" contains error.",
                    docIdStr))
        {
        }
    }
}
